src/environments/environment.staging.ts

```diff
@@ -0,0 +1,35 @@
+import { Profile } from '../app/auth/profiles/profiles.component';
+
+export const AppConfig = {
```

Comment:
We have the staging and production environment now, which differ in their client ID.

To define the staging environment, let's:
- import  the AppConfig from `environment.prod.ts` into `environment.staging.ts`
-  override the client ID
-  export the modified AppConfig

We can remove the need for duplicating the entire `AppConfig` this way

Reply:
I see, alright!